auto_dev/commands/augment.py

```diff
@@ -466,8 +490,21 @@ def skill_from_fsm(spec_file: str, skill_public_id: PublicId, auto_confirm: bool
 
     output = scaffolder.scaffold()
     logger.info(f"Skill scaffolded: {output}")
-
-    # We now need to update the behvaiours.py file with the new handler
+    write_to_file(behaviour_path, output, FileType.PYTHON)
+    logger.info(f"Behaviours file updated: {behaviour_path}")
+    class_name = f"{snake_to_camel(scaffolder.fsm_spec.label).capitalize()}FsmBehaviour"
+    skill_yaml = read_from_file(skill_dir / "skill.yaml", FileType.YAML)
+    if "behaviours" not in skill_yaml:
+        skill_yaml["behaviours"] = {}
+    skill_yaml["behaviours"]["main"] = {
+        "args": {},
+        "class_name": class_name,
+    }
+    write_to_file(skill_dir / "skill.yaml", skill_yaml, FileType.YAML)
+    logger.info(f"Skill.yaml updated: {skill_dir / 'skill.yaml'}")
+    package_manager = PackageManager(verbose=verbose)
+    package_manager.fingerprint_component(PackageType.SKILL, skill_public_id)
```

Comment:
This shouldn't be here, should be in the create from fsm workflow in create

Reply:
TBH i think that someone augmenting won't know to update that file, we should do it here, or at the very least have an option to

Reply:
then when they run it wioll say they should, bear in mind, this gets run as part of eject.

At no point do we enforce consistence up until build at which point we go ahead an d already lock etc